platform/keyword-registry/src/main/java/org/codice/ddf/keyword/KeywordRegistryImpl.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.keyword;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class KeywordRegistryImpl implements KeywordRegistry {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(KeywordRegistryImpl.class);
+
+  private List<String> keywordList = Collections.emptyList();
+
+  private Map<String, String> mappings = Collections.emptyMap();
+
+  @Override
+  public Optional<String> getKeyword(String keyword) {
+    return Optional.ofNullable(mappings.get(keyword));
+  }
+
+  @Override
+  public Map<String, String> getKeywords() {
+    return mappings;
+  }
+
+  public void setKeywordList(List<String> keywordList) {
```

Comment:
✏️ blueprint/metatype methods should use `@SuppressWarnings("WeakerAccess" /* public method needed for blueprint/metatype */)`